### DISCUSSION TITLE
希望能支持lrc格式的逐字歌词

### DISCUSSION THREAD

**Discussion by wyllwyc** [amll-dev/applemusic-like-lyrics#290] [Ideas]
https://github.com/chenmozhijin/LDDC该项目可以获取lrc格式的逐字歌词

**Comment by MianSoft** [comment]
AMLL 支持旧版 ESLrc 格式(新旧版区别:[问题反馈 歌词格式不通用 #51](https://github.com/ESLyric/feedback/issues/51))，从这个项目仓库的预览图片来看支持导出 ESLrc 格式的歌词,应该是可以使用的
扩展型 LyRiC 的话目前暂不支持

**Comment by planegenes** [comment]
可以下载为ASS格式歌词使用脚本转为ttml，需要使用到Aegisub

https://github.com/ranhengzhang/amll-ttml-db-raw-data/blob/main/ass2ttml.lua
